### ISSUE TITLE
compile error  HELP !

### ISSUE THREAD

**Issue by lilyswang** [AlexeyAB/darknet#7653]
If something doesn’t work for you, then show 2 screenshots:
1. screenshots of your issue
![image](https://user-images.githubusercontent.com/30424943/116229489-264ef900-a789-11eb-99a0-b0a10eb39a14.png)

2. my configuration:

![image](https://user-images.githubusercontent.com/30424943/116229736-7201a280-a789-11eb-92d5-4b7e713616a6.png)

#define CUDNN_MAJOR 7
#define CUDNN_MINOR 6
#define CUDNN_PATCHLEVEL 5

**Comment by Krogsager** [comment]
What is your
 - OS
 - makefile
 - output of `nvidia-smi`
 - do you have cuda in your PATH?
 - did you follow the pre/post installation steps for installing CUDA toolkit?

**Comment by lilyswang** [comment]
Hi @Krogsager 👍 
ubuntu 18.04
makefile:
![image](https://user-images.githubusercontent.com/30424943/116233523-fb1ad880-a78d-11eb-8eb8-682f61adbcdb.png)

![image](https://user-images.githubusercontent.com/30424943/116233585-0f5ed580-a78e-11eb-9eea-ce21735fa0e9.png)

i have add cuda 10.0 in my path

**Comment by Krogsager** [comment]
Did you set the correct compute capability in the Makefile, in corrospondance with your Tesla GPU?

**Comment by lilyswang** [comment]
![image](https://user-images.githubusercontent.com/30424943/116235311-446c2780-a790-11eb-8a7f-29bb58fd70ca.png)
emm..    change the Makefile like this ,but  the problem is also exist.

**Comment by Krogsager** [comment]
Please share the command you ran, and the terminal output. Copy-paste instead of posting screenshots, which can be hard to read and impossible to search in.
Did you `make clean` before running make?

**Comment by stephanecharette** [comment]
Are you using an old version of Darknet?  Because you know that CUDA 10.0 won't work in modern Darknet.  Though not for the reasons you're seeing above.  The errors you are seeing makes me think you don't have CUDA/cuDNN installed correctly.

**Comment by ekesdf** [comment]
yea reinstall cuda 11.0 and cudnn8.x and it will work

**Comment by lilyswang** [comment]
Thanks @stephanecharette ， @ekesdf  I  will try it !

**Comment by lilyswang** [comment]
@stephanecharette ,yes I use the old version, maybe the cuda is not installed  correctly.

**Comment by aparico** [comment]
I'm trying to so this in Colab using their GPU and I'm having similar error:

```
collect2: error: ld returned 1 exit status
Makefile:141: recipe for target 'darknet' failed
make: *** [darknet] Error 1
```

**Comment by ekesdf** [comment]
Can you Upload your make File aß an txt File PLZ thy

**Comment by aparico** [comment]
> Can you upload your make File as a txt File plz thy

[Here you go](https://colab.research.google.com/drive/1mzL6WyY9BRx4xX476eQdhKDnd_eixBlG?usp=sharing). I suspect it's related to the CUDA version. I have used these codes a month ago and it was working well.

**Comment by ekesdf** [comment]
Hi in the cell where they write makefile I just changed the line **LIBSO=1** to **LIBSO=0** and it worked like a charm

**Comment by aparico** [comment]
> Hi in the cell where they write makefile I just changed the line **LIBSO=1** to **LIBSO=0** and it worked like a charm

I'm still having the same error. :(

**Comment by ekesdf** [comment]
Rly when just loaded up the notebook and changed this line and run it again it build it with out any error 🤔

**Comment by ekesdf** [comment]
Do you run this notebook locally or via google colab??

**Comment by aparico** [comment]
> Do you run this notebook locally or via google colab??

I ran it in Colab. In the link I provided, a different repo for darknet was cloned. If I clone AlexeyAB's repo, the errors show up.

**Comment by ekesdf** [comment]
Wait so you send me the wrong notebook

**Comment by ekesdf** [comment]
But you download the AlexyAB repo 

````#download the newly released yolov4 ConvNet weights
%cd /content/darknet
!wget https://github.com/AlexeyAB/darknet/releases/download/darknet_yolo_v3_optimal/yolov4.conv.137````

**Comment by aparico** [comment]
> Wait so you send me the wrong notebook

Let me share you [what I exactly tried](https://colab.research.google.com/drive/1fIK6g2oYyMrNheEmtVObWgU1JMrOv_e8?usp=sharing )

**Comment by ekesdf** [comment]
Wrong make file  just use this [notebook](https://colab.research.google.com/drive/1fIK6g2oYyMrNheEmtVObWgU1JMrOv_e8?usp=sharing) 
edit so it should work now, but you could just ignore the write make file cell you can just edit by your self

**Comment by aparico** [comment]
> Wrong make file just use this [notebook](https://colab.research.google.com/drive/1fIK6g2oYyMrNheEmtVObWgU1JMrOv_e8?usp=sharing)
> edit so it should work now, but you could just ignore the write make file cell you can just edit by your self

It worked. Thanks!!
